EIPS/eip-99999.md

```diff
@@ -0,0 +1,57 @@
+---
+eip: 99999
```

Comment:
```suggestion
eip: 8014
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.